### ISSUE TITLE
fix asap!

### ISSUE THREAD

**Issue by xAqronHue** [poggit-orphanage/FactionsPro#52]
i can open enemy factions fence gates meaning i can basically walk into there base, make it so i cant open enemy faction fence gates that r in their claim.

**Comment by Esch16** [comment]
Yeah that's because there is no interact handling with PlayerInteractEvent, therefore you can touch/trigger stuff but you cannot break/place blocks. it should be fixed

**Comment by DenielWorld** [comment]
People would still be able to glitch in. The only solution I see here is freezing the player for 1-2 seconds upon an attempt to interact with an enemy base in any way.
